en/identity-server/next/docs/references/tutorials/protect-apps-with-identity-gateway/index.md

```diff
@@ -0,0 +1,2 @@
+
+{% include "../../../../7.1.0/docs/references/tutorials/protect-apps-with-identity-gateway/index.md" %}
```

Comment:
includes should be in `en/includes` folder, so including markdown files from 7.1 is an anti pattern. Instead let's move the content to `includes` folder and reference it from both 7.1 and next docs.